src/python/WMComponent/PhEDExInjector/PhEDExInjectorPoller.py

```diff
@@ -92,6 +148,33 @@ def setup(self, parameters):
 
         return
 
+    def algorithm(self, parameters):
+        """
+        _algorithm_
+
+        Poll the database for uninjected files and attempt to inject them into
+        PhEDEx.
+        """
+        logging.info("Running PhEDEx injector poller algorithm...")
+
+        self.pollCounter += 1
+
+        if self.blocksToRecover:
+            logging.info("""PhEDExInjector Recovery:
```

Comment:
I see. Since you are touching this code, it would be better to make these cosmetic fixes inside this PR. Of course, I see no reason why not to make a new one for those changes as well.